src/flake8_requirements/checker.py

```diff
@@ -252,6 +252,9 @@ class Flake8Checker(object):
     # User defined project->modules mapping.
     known_modules = {}
 
+    # max depth to resolve recursive requirements
+    requirements_max_depth = 0
```

Comment:
Maybe allow one recursion by default. Then it might work for 95% of cases out of the box. What do you think?